packages/itwin/measure-tools/package.json

```diff
@@ -0,0 +1,130 @@
+{
+  "name": "@itwin/measure-tools-react",
+  "version": "0.1.0",
+  "license": "MIT",
+  "description": "Frontend framework and tools for measurements",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/iTwin/viewer-components-react/tree/master/packages/itwin/measure-tools"
+  },
+  "main": "lib/cjs/measure-tools-react.js",
+  "module": "lib/esm/measure-tools-react.js",
+  "typings": "lib/cjs/measure-tools-react",
+  "scripts": {
+    "build": "npm run copy:public && npm run -s build:cjs && npm run -s build:esm && npm run webpack:tests",
+    "build:cjs": "tsc 1>&2 --outDir lib/cjs && npm run -s copy:cjs",
+    "build:esm": "tsc 1>&2 --outDir lib/esm --module ES2020 && npm run -s copy:esm",
+    "copy:public": "cpx \"./public/**/*\" ./lib/public",
+    "copy:cjs": "cpx \"./src/**/*.*css\" ./lib/cjs",
+    "copy:esm": "cpx \"./src/**/*.*css\" ./lib/esm",
+    "pseudolocalize": "betools pseudolocalize --englishDir ./public/locales/en --out ./lib/public/locales/en-PSEUDO",
+    "watch": "tsc --w",
+    "clean": "rimraf lib .rush/temp/package-deps*.json",
+    "rebuild": "npm run clean && npm run build",
+    "lint": "eslint -f visualstudio \"./src/**/*.{ts,tsx}\" 1>&2",
+    "test": "certa -c certa.json -r chrome",
+    "cover": "nyc npm test",
+    "madge": "madge --circular --extensions ts,tsx src",
+    "webpack:tests": "webpack --config webpack.config.js 1>&2"
+  },
+  "keywords": [
+    "Bentley",
+    "BIM",
+    "iModel",
+    "iTwin",
+    "Measurement",
+    "Measure",
+    "iTwin.js"
+  ],
+  "author": {
+    "name": "Bentley Systems, Inc.",
+    "url": "http://www.bentley.com"
+  },
+  "files": [
+    "lib/**/*",
+    "!lib/**/test/**/*"
+  ],
+  "peerDependencies": {
+    "@itwin/appui-abstract": "rc",
+    "@itwin/appui-layout-react": "rc",
+    "@itwin/appui-react": "rc",
+    "@itwin/components-react": "rc",
+    "@itwin/core-bentley": "rc",
+    "@itwin/core-common": "rc",
+    "@itwin/core-frontend": "rc",
+    "@itwin/core-geometry": "rc",
+    "@itwin/core-quantity": "rc",
+    "@itwin/core-react": "rc",
+    "@itwin/core-telemetry": "rc",
```

Comment:
Does telemetry need to be a per dep? We can follow up on this but did want to point it out.

Reply:
I believe we're just using the type in this PKG so it can be a direct dep. Will update and trim deps in a separate pr